### ISSUE TITLE
Tutorial.md says to use "host.docker.local" instead of "host.docker.internal"

### ISSUE THREAD

**Issue by andydude** [tektoncd/pipeline#2970]

**Comment by pritidesai** [comment]
It's under the [Prerequisites](https://github.com/tektoncd/pipeline/blob/master/docs/tutorial.md#prerequisites) section, replace 

```
Set host.docker.local:5000 as an insecure registry with Docker for Desktop. See the Docker insecure registry documentation. for details.
```

With:

```
Set host.docker.internal:5000 as an insecure registry with Docker for Desktop. See the Docker insecure registry documentation. for details.
```

/kind documentation

**Comment by juanri0s** [comment]
I can work on this! I'll PR by end of day

**Comment by dlorenc** [comment]
I think this was fixed with #2993!
